### PR TITLE
docs(eap): added docker registry accounts

### PR DESCRIPTION
**Pull request by nemesisOsorio**

### CODE REVIEW

File: content/en/armory-enterprise/plugin-guide/plugin-external-account.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Any sidecar can populate the accounts directory by pulling information from othe
 |------------------------|-----------------------------------------|
 | Armory 2.23 (OSS 1.23) | Kubernetes and Cloudfoundry implemented |
 | OSS 1.24               | AWS and ECS implemented                 |
+| Armory 2.28 (OSS 1.28) | Docker implemented                      |
```
Comment (nemesisOsorio) [inline content/en/armory-enterprise/plugin-guide/plugin-external-account.md:41]: after armory LTS release we can merge or update it

File: content/en/armory-enterprise/plugin-guide/plugin-external-account.md [hunk 2]
```diff
@@ -311,6 +323,10 @@ This plugin can read account credentials in the following layouts:
 
 ## Release Notes
 
+* v0.3.0 (01/30/2022)
```
Comment (nemesisOsorio) [inline content/en/armory-enterprise/plugin-guide/plugin-external-account.md:344]: same here, after the plugin release we can update it

**Review by german-muzquiz — APPROVED** [review]

File: content/en/armory-enterprise/plugin-guide/plugin-external-account.md (hunk 2 above)
Comment (nemesisOsorio) [reply content/en/armory-enterprise/plugin-guide/plugin-external-account.md:344]: we already created the release https://github.com/armory-plugins/external-accounts/releases/tag/v0.3.0

File: content/en/armory-enterprise/plugin-guide/plugin-external-account.md (hunk 1 above)
Comment (nemesisOsorio) [reply content/en/armory-enterprise/plugin-guide/plugin-external-account.md:41]: the changes are ready for those versions

**Review by sergio-quintero — APPROVED** [review]

**Review by link108 — APPROVED** [review]

**Review by link108 — APPROVED** [review]

**Pull request merged**